scripts/mini_hp.sh

```diff
@@ -10,6 +10,7 @@ TARGET_TERMS=(
 'HP:0012773|BOT' #|Reduced upper to lower segment ratio
 'HP:0041249|BOT' #|Fractured nose
 'HP:0010533|BOT' #|Spasmus nutans
+'HP:0002090|BOT' #|Pneumonia
```

Comment:
Why do we need another Phenotypic abnormality? We already have 3 and all of their ancestors.